kalite/topic_tools/tests/annotate_tests.py

```diff
@@ -0,0 +1,82 @@
+
+from kalite.topic_tools.annotate import update_content_availability
+from kalite.testing.base import KALiteTestCase
+from django.test import TestCase
+from kalite.topic_tools.content_models import Item, annotate_content_models, set_database, unparse_model_data
+from . import settings 
+from peewee import Using
+from playhouse.shortcuts import model_to_dict
+import json
+
+class AnnotateTestCase(TestCase):
+  TITLE = "testing "
+  AVAILABLE = False
+  KIND = "Exercise"
+  
+  @set_database
+  def setUp(self, db=None):
```

Comment:
Your text editor keeps changing its indentation. Now you're using 2 spaces here. Before you were using 8 spaces. Can you sort this out and set this to 4 spaces?